projects/ngx-darkbox-gallery/src/lib/ngx-darkbox-gallery.component.ts

```diff
@@ -70,10 +87,13 @@ export class NgxDarkboxGalleryComponent implements OnInit, OnChanges {
   @Output()
   darkboxImageLoaded = new EventEmitter<Image>();
 
-  constructor() { }
+  constructor(private configurationService: ConfigurationService) { }
 
   ngOnInit(): void {
     this.initializeConfiguration(this.configuration);
+    if (this.loadMoreImagesEvents) {
```

Comment:
We should react to a change on our input in the onChanges method and unsubscribe / resubscribe in the case of change of this property.

Reply:
Unfortunately I am not quite sure what exactly you mean. These are events that come from outside of the library as an `Observable`. Because the changes are done on the `Observable` unsubscribing / resubscribing is not necessary in my opinion. If you have a better idea on how to react to outside events please let me know in more detail.

Reply:
The problem is that the observable is injected as an input and this input may change during the livetime of our component. So we have to react to those changes in the onChanges method and unsubscribe from the old and subscribe to the potentially new observable. Subscribing onInit and unsubscribing onDestroy is not enough

Reply:
I changed the event subscription in cf03aa1. The issue with the placeholder still has to be solved. On the static grid all images get replaced by placeholders and on the fluid one all disappear while loading another batch.

Reply:
I was able to fix the placeholders not being displayed on the fluid grid, but I still do not have a good solution for the problem, that all images are replaced by the placeholder while loading the next batch.

Reply:
As discussed I opened an issue for this (#138). Therefore I think this PR is now ready to be merged.